### PR TITLE
Web console: fix problems with display route and route warnings

### LINKED ISSUE #7720: Overview route warnings, but no warnings on routes page
I get a warning about routes on the overview page ...

openshift_web_console

... but no routes have a warning on the routes page.

openshift_web_console

@jwforres

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #7720

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres PTAL

File: assets/app/scripts/filters/resources.js [hunk 1]
```diff
@@ -926,11 +926,24 @@ angular.module('openshiftConsole')
...
+          admittedConditions.push(admitted);
+        }
+      });
+
+      if (!admittedConditions.length) {
         return "Unknown";
       }
-      return admittedCondition.status === "True" ? "Admitted" : "Rejected";
+
+      if (_.some(admittedConditions, { status: "True" })) {
```
Comment (jwforres) [inline assets/app/scripts/filters/resources.js]: trying to decide if you should consider a partially admitted route as admitted or rejected, or if it deserves its own "PartiallyAdmitted" status

**Comment by jwforres** [comment]
just the one comment, otherwise LGTM

File: assets/app/scripts/filters/resources.js (hunk 1 above)
Comment (spadgett) [inline assets/app/scripts/filters/resources.js]: I think the problem is we're trying to show multiple status as one value. We're only using this for preferred display routes. I suggest we just move the logic to route service.

Comment (jwforres) [inline assets/app/scripts/filters/resources.js]: Yeah I agree, when I originally added it I was using it somewhere else but
there is no point in it being its own filter now.

**Comment by spadgett** [comment]
@jwforres another look?

**Comment by jwforres** [comment]
LGTM, I doubt having a partially admitted route is going to be common, let alone partially admitted routes AND having multiple routes, so shortcutting the "admitted" comparison is probably fine as a performance optimization

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
[merge]

**Comment by deads2k** [comment]
@stevekuznetsov Check `oc status` to make sure we have an indicator on routes to show if that route isn't honored.

**Pull request merged**